app/src/main/res/layout/activity_adopt_activity.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".activity_adoptPet">
+
+    <android.support.v4.view.ViewPager
+        android:id="@+id/swipingpicture"
+        android:layout_width="match_parent"
+        android:layout_height="276sp" />
+
+    <ImageButton
+        android:id="@+id/likeBtn"
+        android:layout_width="112dp"
```

Comment:
No hardcoded size

Reply:
I'll stop pointing out each instance both here and as to naming convention to avoid being obnoxious